Fix OpenAI client usage example for embeddings (#720)
huggingface/text-embeddings-inference: docs/source/en/quick_tour.md

```

#### OpenAI

> The server is availabe at : localhost:8080/v1/embeddings

Reviewer: ```suggestion
```

> You can install it via pip as `pip install --upgrade openai`, and then run:

Reviewer: ```suggestion
To send requests to the [OpenAI Embeddings API](https://platform.openai.com/docs/api-reference/embeddings/create) exposed on Text Embeddings Inference (TEI) with the OpenAI Python SDK, you can install it as `pip install --upgrade openai`, and then run the following snippet:
```

```python
import os
from openai import OpenAI

> client = OpenAI(base_url="http://localhost:8080/v1", api_key= "fake")

Reviewer: ```suggestion
client = OpenAI(base_url="http://localhost:8080/v1", api_key= "-")
```

response = client.embeddings.create(

>   model="tei",
>   input="What is deep learning?"

Reviewer: ```suggestion
    model="text-embeddings-inference",
    input="What is Deep Learning?",
```

)

print(response.data[0].embedding)
```

> 

Reviewer: Maybe also add `curl` example? Note that the closing backticks are missing another one, but GitHub won't let me add that and still be compliant with the suggestion format, so you'll need to add it afterwards 🤗 

```suggestion

Alternatively, you can also send the request with cURL as follows:

```bash
curl http://localhost/v1/embeddings \
  -H "Content-Type: application/json" \
  -d '{
    "input": "What is Deep Learning?",
    "model": "text-embeddings-inference",
    "encoding_format": "float"
  }'
``

```
